src/MoBi.UI/Views/EditParametersInContainerView.cs

```diff
@@ -371,6 +374,9 @@ public string ParentName
          set => lblParentName.Text = value.FormatForLabel(checkCase: false);
       }
 
+      public void SetNameForContainer(IContainer container) =>
+         lblParentName.Text = string.IsNullOrEmpty(container.Name) ? $"New {_typeResolver.TypeFor(container)}".FormatForLabel(checkCase: false) : container.Name.FormatForLabel(checkCase: false);
+
```

Comment:
Let's put the conditional in the presenter and no matter what string comes in, have the view call `FormatForLabel`

Reply:
Once you move it, you can add a test for the presenter that checks if the name is correctly set in the view.